docs/1manifoldLearningCircles.md

```diff
@@ -0,0 +1,47 @@
+# 1-Manifold Learning Circles example
+
+![1-Manifold Learning Circles  example Image](https://topology-tool-kit.github.io/img/gallery/1manifoldLearningCircles.jpeg)
+
+## Pipeline description
+
+
+This example first loads a point cloud from disk. The points are arranged in mainly two concentric circles, one bigger than the other.
+
+In a pre-processing, the data is converted to a format understandable by Paraview using the TableToPoints filter (upper left view, above screenshot). Gaussian Resampling is applied to the data. This filter has the effect of injecting input points to a structured data. For each injection, each point will "splat", or distribute values to nearby vertices. Only a slice of the data is kept (upper right, above screenshot). The data set is now represented by a density scalar field in 2D. 
```

Comment:
could you address the list of remarks I made in one of your previous PRs?
https://github.com/topology-tool-kit/ttk-data/pull/92

(for instance TableToPoints -› `TableToPoints`, etc.)